tmt/steps/prepare/shell.py

```diff
@@ -30,28 +33,28 @@ class PrepareShell(tmt.steps.prepare.PreparePlugin):
     _keys = ["script"]
 
     @classmethod
-    def options(cls, how=None):
+    def options(cls, how: Any = None) -> Any:
         """ Prepare command line options """
         return [
             click.option(
                 '-s', '--script', metavar='SCRIPT',
                 help='Shell script to be executed.')
             ] + super().options(how)
 
-    def default(self, option, default=None):
+    def default(self, option: str, default: Optional[Any] = None) -> Any:
         """ Return default data for given option """
         if option == 'script':
             return []
         return default
 
-    def wake(self, keys=None):
+    def wake(self, keys: Optional[List[str]] = None) -> None:
         """ Wake up the plugin, process data, apply options """
         super().wake(keys=keys)
 
         # Convert to list if single script provided
         tmt.utils.listify(self.data, keys=['script'])
 
-    def go(self, guest):
+    def go(self, guest: 'Guest') -> None:
```

Comment:
You don't need string wrap when a class can be imported in runtime, which si the case here. No special care needed.